wallet/udb/addressmanager_test.go

```diff
@@ -1033,3 +1019,14 @@ func TestManager(t *testing.T) {
 	}
 	testManagerAPI(tc)
 }
+
+func TestMain(m *testing.M){
+	err := createEmptyDB()
+	defer os.RemoveAll(filepath.Dir(dbPath))
```

Comment:
Since we're doing cleanup here, let's also do the path creation as well.  It's not clear at all what dbPath is, where it was created, or why we are deleting it.